tools/syz-declextract/clangtool/declextract.cpp

```diff
@@ -447,10 +448,16 @@ const T* Extractor::findFirstMatch(const Node* Expr, const Condition& Cond) {
   return Matches.empty() ? nullptr : Matches[0];
 }
 
+// Extracts the first function reference from the expression.
```

Comment:
Why exactly the first? If it's a const expression, is it not yet evaluated at the moment we do the processing?

Reply:
Added a TODO:

```
// TODO: try to extract the actual function reference the expression will be evaluated to
// (the first one is not necessarily the right one).
```

I dunno if there is an easy way to do such evaluation in clang (possibly). It's also not super important as these tricky expressions come up only for debug fs. Normal drivers just assign a function.